src/compiler/diagnosticMessages.json

```diff
@@ -7135,6 +7135,18 @@
         "category": "Message",
         "code": 95169
     },
+    "Remove unused parameter '{0}'": {

```

Comment:
Since the codefix changes the tag, the message should refer to `'@param' tag` in the same way that the error does.